web-app/src/components/Header/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import { Navbar, Nav, NavItem } from 'react-bootstrap';
+import 'react-bootstrap';
+import LoginService from '../../pages/Login/LoginService';
+import styles from './style.css';
+
+class Header extends React.Component {
+    constructor(props){
+        super(props);
+        this.handleLogout = this.handleLogout.bind(this);
+        this.onHomeClick = this.onHomeClick.bind(this);
+        this.onSettingsClick = this.onSettingsClick.bind(this);
+
+        this.Auth = new LoginService();
+    }
+
+// TODO: Get email from JWT.
+
+    handleLogout(){
+        this.Auth.logout()
+        this.props.history.replace('/login');
+    }
+
+    onHomeClick(){
+        console.log("Home clicked");
+      }
+
+    onSettingsClick(){
+        console.log('Settings clicked');
+    }
+
+    render() {
+        return (
+            <div className="App">
+
+                <Navbar inverse collapseOnSelect className={styles.NavBarStyling}>
+                    <Navbar.Header>
+                        <Navbar.Brand>
+                            <a href="#brand">AdminCore</a>
+                        </Navbar.Brand>
+                        <Navbar.Toggle />
+                    </Navbar.Header>
+                    <Navbar.Collapse>
+                        <Nav>
+                            <NavItem eventKey={1}  onClick={this.onHomeClick}>
+                            Home
+                            </NavItem>
+                        </Nav>
+                        <Nav pullRight>
+                            <NavItem eventKey={1}  onClick={this.onSettingsClick}
+                                style={{visibility: this.props.user.employeeRoleId < 3 ? 'visible' : 'hidden' }}>
```

Comment:
I'm not a huge fan of magic numbers (for maintainability), might be better using an enumeration here, especially for readability. An example could be something like:

const roles = {
  admin: 1,
  super: 2,
  standard: 3
};

This check would become this.props.user.employeeRoleId < roles.standard etc.

Don't have to do it, but I think it helps